library/general/test/event_dispatcher_test.rb

```diff
@@ -32,6 +32,14 @@ def user_input
   end
 end
 
+class DispatcherHandleEventTestDialog
+  include UI::EventDispatcher
+
+  def handle_event(_input)
+    finish_dialog(:other)
```

Comment:
maybe just check here that input is other or simple `finish_dialog(input)` so test verify that proper input is passed.